app/src/main/res/layout/fragment_contacts.xml

```diff
@@ -11,14 +11,33 @@
         android:layout_height="wrap_content"
         app:title="@string/title_contacts" />
 
-    <com.yterletskyi.happyfriend.common.view.SearchView
-        android:id="@+id/etSearch"
+    <androidx.constraintlayout.widget.ConstraintLayout
         android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:layout_marginHorizontal="8dp"
-        android:layout_marginVertical="8dp"
-        android:hint="@string/search_hint_contacts"
-        android:inputType="text" />
+        android:layout_height="wrap_content">
+        <com.yterletskyi.happyfriend.common.view.SearchView
+            android:id="@+id/etSearch"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginHorizontal="8dp"
+            android:layout_marginVertical="8dp"
+            android:hint="@string/search_hint_contacts"
+            android:inputType="text"
+            android:maxLength="35"
```

Comment:
This will work in most cases, but we never know the max possible length of a person. In theory some citizens (for example UAE or Spain) have very-very long names -- this will cause a bug for them. 
Let's better not limit the length, but add proper constraints.